Components/List.js

```diff
@@ -1,31 +1,35 @@
 import React from 'react';
-import { Text ,View,SafeAreaView,ScrollView } from 'react-native';
-import { ListItem, Avatar } from 'react-native-elements'
+import { Text ,View,SafeAreaView,ScrollView, TouchableOpacity} from 'react-native';
 import list from './Movielist';
+import ItemView from './ItemView';
 
 
-const ListScreen = ({ navigation, route }) => {
+
+const ListScreen = ({ navigation}) => {
     return (
       <View>
         <Text>Movie Lists</Text>
         <SafeAreaView>
           <ScrollView>
             {
-            list.map((l, i) => (
-            <ListItem key={i} bottomDivider>
-              
-              <Avatar source={{uri: l.avatar_url}} />
-              <ListItem.Content>
-                <ListItem.Title>{l.title}</ListItem.Title>
-                <ListItem.Subtitle>{l.year}</ListItem.Subtitle>
-              </ListItem.Content>
-            </ListItem>
-             ))}
+            list.map((l, i) => {
+              if(l.ping==='0'){
```

Comment:
what iss ping? Can you name it appropriately and what does'0' mean.. everthing has to be named right